packages/legend-graph/src/graphManager/protocol/pure/v1/V1_PureGraphManager.ts

```diff
@@ -769,6 +769,31 @@ export class V1_PureGraphManager extends AbstractPureGraphManager {
         options,
       );
 
+      // run second pass builder for functions only to build the correct functionName
+      await Promise.all(
+        buildInputs.flatMap(async (input) => {
+          await Promise.all(
```

Comment:
I am a bit worried about doing the `secondPass` because it is trying to build and resolve a bunch of things. It is possible that we might not find those elements in graph as we just build out element paths. 
If this is just about the function name can we just resolve function name in second pass of building light graph. I think we would have enough information from protocol model to do that.
@MauricioUyaguari  what do you think?

Reply:
As we discussed offline, we will resolve `returnType + multiplicity` and `parameter type + multiplicity` during buildLightGraph and leave out resolving taggedValue and stereotype.